templates/uimodules/image-medium.html

```diff
@@ -5,7 +5,7 @@
   </div>
   <div class="user-and-title">
     <a href="/user/{{sharedfile_user.name}}" title="{{sharedfile_user.name}}">
-      <img class="avatar--img" src="{{sharedfile_user.profile_image_url()}}" width="20" height="20">
+      <img class="avatar--img" src="{{sharedfile_user.profile_image_url()}}" width="20" height="20" alt="{{sharedfile_user.name}}">
```

Comment:
Same as above (although title here is using `name` not `display_name()`).

Reply:
Should I change to display_name? I'm unclear on what's available where, so I just copied what was in the title

Reply:
I'm not certain, although I prefer consistency. FYI, anywhere you have a user object, you should be able to use `display_name()`. That function returns `full_name`, if assigned or `name` (username) if it isn't. Note that `name` has inherent restrictions on characters-- a-z, numbers, dashes. `full_name` could contain anything, so using `display_name()` should always require an `escape()`; the above usage does not, which means it could have security risk.

Reply:
Okay, updated to use the escaped display name.